server/assets/javascripts/reset-password.js

```diff
@@ -98,6 +102,15 @@ function displayErrors() {
     });
 }
 
+//Used for 'Verification code is required.' message
+$('#main-content').on('click', '#emailVerificationControl_but_verify_code', function() {
+    const infoElem = $('.error.itemLevel.show');
```

Comment:
Might this return > 1 element?

Reply:
Or does it just add the class to each one without requiring a loop?

Reply:
On this page I could only get it to return 1 element through testing diff scenarios, not sure where we'd see more than 1 info msg on this page as it's only 1 input and afaik it's only for required msg

Not actually sure what happens with 2 elements but I can test this